ElectronicObserver/Window/Tools/SortieRecordViewer/SortieDetail/SortieDetailViewModel.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectronicObserver.Common;
+using ElectronicObserver.Data;
+using ElectronicObserver.KancolleApi.Types.ApiGetMember.ShipDeck;
+using ElectronicObserver.KancolleApi.Types.ApiReqBattleMidnight.Battle;
+using ElectronicObserver.KancolleApi.Types.ApiReqBattleMidnight.SpMidnight;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.Battle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.Battleresult;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.BattleWater;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.EachBattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.EachBattleWater;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.EcBattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.EcMidnightBattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.LdAirbattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.MidnightBattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqCombinedBattle.SpMidnight;
+using ElectronicObserver.KancolleApi.Types.ApiReqMap.Models;
+using ElectronicObserver.KancolleApi.Types.ApiReqMap.Next;
+using ElectronicObserver.KancolleApi.Types.ApiReqMap.Start;
+using ElectronicObserver.KancolleApi.Types.ApiReqSortie.Airbattle;
+using ElectronicObserver.KancolleApi.Types.ApiReqSortie.Battle;
+using ElectronicObserver.KancolleApi.Types.ApiReqSortie.Battleresult;
+using ElectronicObserver.KancolleApi.Types.ApiReqSortie.LdAirbattle;
+using ElectronicObserver.KancolleApi.Types.Interfaces;
+using ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Battle;
+using ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Node;
+using ElectronicObserverTypes;
+
+namespace ElectronicObserver.Window.Tools.SortieRecordViewer.SortieDetail;
+
+public class SortieDetailViewModel : WindowViewModelBase
+{
+	public SortieDetailTranslationViewModel SortieDetail { get; }
+	private BattleFactory BattleFactory { get; }
+
+	public DateTime? StartTime { get; set; }
+	public int World { get; }
+	public int Map { get; }
+	private BattleFleets Fleets { get; }
+	private List<IBaseAirCorpsData>? AirBases { get; }
```

Comment:
This is never used

Reply:
Right, moved that into BattleFleets.
Removed.